src/helpers/sanitize.ts

```diff
@@ -1,16 +1,25 @@
 import { UpdateGroundPlaceProperties } from '../types';
 
+/**
+ * @description Helper to remove all `undefined` values from any object.
+ * @param {Record<string, unknown>} object - Any object that is structured with string index.
+ * @returns {void}
+ */
+const removeUndefinedValues = (object: Record<string, unknown>): void => {
+  Object.keys(object).forEach((key: string) => {
+    if (object[key] === undefined) {
+      delete object[key];
+    }
+  });
+};
+
 /**
  * @description This method is used to remove all `undefined` values that can be found on the propertiesToUpdate object.
  * @param {UpdateGroundPlaceProperties} propertiesToUpdate - Properties to update {name, lattitude, longitude}.
  * @returns {void}
  */
 const sanitizeGroundPlacePropertiesToUpdate = (propertiesToUpdate: UpdateGroundPlaceProperties): void => {
-  Object.keys(propertiesToUpdate).forEach((key: string) => {
-    if (propertiesToUpdate[key] === undefined) {
-      delete propertiesToUpdate[key];
-    }
-  });
+  removeUndefinedValues(propertiesToUpdate);
```

Comment:
is this still useful ? why not calling `removeUndefinedValues` directly ? or renaming it to sanitize if its not clear.

Reply:
Indeed! I'm now using only `removeUndefinedValues` method instead `sanitizeGroundPlacesPropertiesToUpdate`